t/geturl_connectivity.pl.in

```diff
@@ -55,35 +58,52 @@ sub run_httpd {
-    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 1 },
```

Comment:
Style nit: Move `curl => 1|0` to the beginning, like this:

```perl
    { curl => 1, ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0 },
```

Reply:
Done